Dockerfile_test

```diff
@@ -4,22 +4,35 @@
 
 FROM node:16-bullseye
 
-RUN apt-get -qq update \
-&& DEBIAN_FRONTEND=noninteractive apt-get -y install \
-    apt-transport-https \
-    curl \
-    unzip \
-    build-essential \
-    python \
-    libcairo2-dev \
-    libgles2-mesa-dev \
-    libgbm-dev \
-    libllvm3.9 \
-    libprotobuf-dev \
-    libxxf86vm-dev \
-    libopengl0 \
-    xvfb \
-&& apt-get clean
+RUN export DEBIAN_FRONTEND=noninteractive \
+  && apt-get -qq update \
+  && apt-get -y --no-install-recommends install \
+      apt-transport-https \
+      curl \
+      unzip \
+      build-essential \
+      python \
+      libcairo2-dev \
+      libgles2-mesa \
+      libgles2-mesa-dev \
+      libgbm-dev \
+      libprotobuf-dev \
+      libegl1 \
+      xvfb \
+      xauth \
+      libopengl0 \
+      libcurl4 \
+      libuv1-dev \
+      libc6-dev \
+      libcap2-bin \
+  && apt-get -y --purge autoremove \
+  && apt-get clean \
+  && rm -rf /var/lib/apt/lists/*
+
+RUN curl http://archive.ubuntu.com/ubuntu/pool/main/libj/libjpeg-turbo/libjpeg-turbo8_2.0.3-0ubuntu1_amd64.deb --output libjpeg-turbo8_2.0.3-0ubuntu1_amd64.deb
```

Comment:
Is there a reason we need to download this manually vs apt-getting it? Is it a versioning thing?

Reply:
It seems like mablibre-gl depends on a version of libjpeg-turbo with legacy-capabilities enabled for version 8 which doesn't seem to be available in the current apt repository and also a libicu version which is not in the repository.

But for the dockerfile i just copied the packages from `Dockerfile`